boards/arm/dragino_lsn50/support/openocd.cfg

```diff
@@ -0,0 +1,12 @@
+source [find board/stm32f0discovery.cfg]
```

Comment:
Is that good for your board?

Reply:
You are correct, how to do that?

Reply:
It is board specific. Does this board have embedded debugger?

Reply:
You can - have a try - with st_nucleo_l1.cfg

Reply:
@ekarlso there is no embedded ST-Link on this board, right? How is it flashed?

Reply:
Done